components/style/themes/default.less

```diff
@@ -461,3 +461,11 @@
 @slider-dot-border-color-active:      tint(@primary-color, 50%);
 @slider-disabled-color:               @disabled-color;
 @slider-disabled-background-color:    @component-background;
+
+// Collapse
+// ---
+@collapse-header-padding:             12px 0 12px 40px;
+@collapse-header-background-color:    @background-color-light;
+@collapse-content-padding:            0 @padding-md;
+@collapse-content-background-color:   @component-background;
+@collapse-content-box-padding:        @padding-md 0 @padding-md 0;
```

Comment:
`@collapse-content-box-padding` and `collapse-content-padding` are a little confused. Just use a `@collapse-padding: @padding-md` and use it in this way:

```
  & > &-box {
  padding-top: @collapse-padding;
  padding-bottom: @collapse-padding;
}
```